pkg/utils/utils.go

```diff
@@ -70,3 +76,13 @@ func SortByIndex[T appsv1.StatefulSet | corev1.Pod | int](indexFunc func(T) (int
 
 	return result
 }
+
+// GetOperatorNamespace will pick the namespace based on the serviceaccount
+func GetOperatorNamespace() string {
```

Comment:
Is doing this a common pattern? what other controllers does this?

Reply:
https://github.com/kubernetes-sigs/kueue/blob/8f51174d9d018f85fc09c3ab72769f5a7e7ff097/apis/config/v1beta1/defaults.go#L55

Kueue and Jobset both use this for the default namespace on the component config.